src/Query/Grammar.php

```diff
@@ -142,39 +150,31 @@ protected function whereInUnnest(Builder $query, $where)
     }
 
     /**
-     * @param Builder $query
      * @param array{ tokens: string, query: string, options: array<string, scalar> } $where
-     * @return string
      */
-    protected function whereSearchFullText(Builder $query, array $where): string
+    protected function whereSearchFullText(Builder $builder, array $where): string
     {
         return $this->buildSearchFunction('search', $where);
     }
 
     /**
-     * @param Builder $query
      * @param array{ tokens: string, query: string, options: array<string, scalar> } $where
-     * @return string
      */
-    protected function whereSearchNgrams(Builder $query, array $where): string
+    protected function whereSearchNgrams(Builder $builder, array $where): string
     {
         return $this->buildSearchFunction('search_ngrams', $where);
     }
 
     /**
-     * @param Builder $query
      * @param array{ tokens: string, query: string, options: array<string, scalar> } $where
-     * @return string
      */
-    protected function whereSearchSubstring(Builder $query, array $where): string
+    protected function whereSearchSubstring(Builder $builder, array $where): string
     {
         return $this->buildSearchFunction('search_substring', $where);
     }
 
     /**
-     * @param string $function
      * @param array{ tokens: string, query: string, options: array<string, scalar> } $where
-     * @return string
```

Comment:
Please don't delete the return statements.

`@return string`

Reply:
If add type only return statement, PHPStan will warn me, so I want to either declare all of them or only the ones that need additional type information. Which would be better?

Reply:
In the meantime, I'll try to declare everything as it was originally.

Reply:
Yes. Just don't change formatting.